experiments/billable/src/env.ts

```diff
@@ -1,29 +0,0 @@
-import { z } from 'zod'
-
-
-let ENV: ParsedEnv | undefined
-
-// define zod schema for env-vars
-export const envSchema = z.object({
-  RESEND_API_KEY: z.string(),
-  APP_URL: z.string(),
-});
-
-export type ParsedEnv = z.infer<typeof envSchema>;
-
-export function setupEnv(env: Env) {
-  const parsedEnv = envSchema.safeParse(env);
-    if (!parsedEnv.success) {
-      throw new Error(`Invalid environment variables: ${parsedEnv.error.message}`);
-    }
-    ENV = parsedEnv.data;
-}
-
-export function getEnv(): ParsedEnv {
-  if (!ENV) {
-    throw new Error("ENV is undefined.")
-  }
-  return ENV
-}
-
-
```

Comment:
@peterp I removed this since it would take some thinking to figure out how we account for this ^ in the api. e.g. do we have zod schema for env. If we do, wondering why do we need it if cloudflare already gens the types for the env vars with its `Env` global type.

That ^, and the fact that the router provides the env as context, made me think that maybe its fine to do without it for now, but wanted to ask you about it.